web/src/components/tasks/business-formation/business/FormationDate.tsx

```diff
@@ -30,22 +31,27 @@ export const FormationDate = (props: Props): ReactElement => {
   const { state, setFormationFormData, setFieldsInteracted } = useContext(BusinessFormationContext);
   const { doesFieldHaveError } = useFormationErrors();
   const dateFormat = "MM/DD/YYYY";
+  const direction = isForeignCorporation(state.formationFormData.legalType) ? "column" : "row";
```

Comment:
This might be the best solution here but I'm unsure why the legal type should change the direction of the grid

Reply:
Yea its hard to know when the MM/DD/YYYY label in the formation date should be underneath it without knowing what the business designator secondary text is and it only changes during foreign corporations that it is shown under the effective date

Reply:
explained to sam over zoom